consensus/src/member.rs

```diff
@@ -700,5 +683,19 @@ pub async fn run_session<
         debug!(target: "AlephBFT-member", "{:?} Network stopped.", index);
     }
 
+    if !runway_handle.is_terminated() {
+        if let Err(()) = runway_handle.await {
+            warn!(target: "AlephBFT-member", "{:?} Runway task stopped with an error", index);
+        }
+        debug!(target: "AlephBFT-member", "{:?} Runway stopped.", index);
+    }
```

Comment:
Suggestion: consider extracting this as a function now? It's repeated 5-6 times, and I think it would be really nice to start chipping away at these multi-hundred-line functions that setup the components.